src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -427,12 +448,15 @@ function isInsufficientBalance(
     return false
   }
   const balanceToFormat = formatTokenValue(balance, token.decimals)
-  const nativeBalanceFormatted = formatTokenValue(nativeBalance, token.decimals)
   if (network === BlockchainEnum.NEAR) {
     if (isBaseToken(token) && token.address === "wrap.near") {
-      return formAmount >= balanceToFormat + nativeBalanceFormatted
+      const balanceToFormat = formatTokenValue(
+        balance + nativeBalance,
+        token.decimals
+      )
+      return formAmount > balanceToFormat
     }
-    return formAmount >= balanceToFormat
+    return formAmount > balanceToFormat
```

Comment:
DIdn't pay to attention to it previously, but you should not compare strings, as you can get unexpected result. For example, `"10" > "9"`will give false.

Reply:
Good catch, thanks!